docs/eventing/sources/ping-source/index.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "PingSource"
+weight: 31
+type: "docs"
+aliases:
+  - /docs/eventing/sources/pingsource
+  - ../cronjob-source
+  - /docs/eventing/samples/ping-source
+---
+
+![version](https://img.shields.io/badge/API_Version-v1beta2-red?style=flat-square)
+
+A PingSource is an event source that produces events with a fixed payload on a specified [cron](https://en.wikipedia.org/wiki/Cron) schedule.
+
+The following example shows how you can configure a PingSource as an event source that sends events every minute to a Knative service named `event-display` that is used as a sink.
+
+## Before you begin
+
+1. To create a Knative service to use as a sink, you must install [Knative Serving](../../../install).
```

Comment:
Personally I would prefer to not have to install Serving just to try this example.

Reply:
Is that possible though? I.e. how would someone use or test a PingSource is working w/o a service?

Reply:
The current doc shows how to use a k8s service: https://knative.dev/docs/eventing/sources/pingsource/#creating-the-event-display-service